src/languageServerInstaller.ts

```diff
@@ -17,11 +22,13 @@ export class LanguageServerInstaller {
 			isInstalled = true;
 		} catch (err) {
 			// TODO: verify error was in fact binary not found
+			this.reporter.sendTelemetryException(err);
```

Comment:
Is there a way of knowing where was an exception raised from or add a flag somehow? For example if we end up looking at generic filesystem exceptions, a file not found can be because it was never written/installed, or because it was removed after installation so it might be helpful to know which case it is.

Reply:
Oh I guess we'd see that hopefully from the trace. 💡